pages/docs/tutorials/streetlights.md

```diff
@@ -1,244 +1,263 @@
-      message:
-        name: LightMeasured
-        payload:
-          type: object
-          properties:
-            id:
-              type: integer
-              minimum: 0
-              description: Id of the streetlight.
-            lumens:
-              type: integer
-              minimum: 0
-              description: Light intensity measured in lumens.
-            sentAt:
-              type: string
-              format: date-time
-              description: Date and time when the message was sent.
-EOT`}
-</CodeBlock>
-
-### 4. Trigger generation of the Node.js code:
-<CodeBlock language="bash">
-{`ag asyncapi.yaml @asyncapi/nodejs-template -o output -p server=mosquitto`}
-</CodeBlock>
-
-### 5. And voilà! List all files in directory and check that Node.js application is generated:
-<CodeBlock language="bash">
-{`cd output && ls`}
-</CodeBlock>
-
-# Running your code
-
-### 1. Install dependencies of newly generated application:
-<CodeBlock language="bash">
-{`npm install`}
-</CodeBlock>
-
-### 2. Start the application:
-<CodeBlock language="bash">
-{`npm start`}
-</CodeBlock>
-
-### 3. In another terminal install the MQTT.js library:
-<CodeBlock language="bash">
-{`npm install mqtt -g`}
-</CodeBlock>
-
-### 4. Send a correct message to your application:
-<CodeBlock language="bash">
-{`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": 3, "sentAt": "2017-06-07T12:34:32.000Z"}'`}
-</CodeBlock>
-
-### 5. Send an incorrect message to your application:
-<CodeBlock language="bash">
-{`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": "3", "sentAt": "2017-06-07T12:34:32.000Z"}'`}
-</CodeBlock>
-
-### 6. Go back to the previous terminal and check if your application logged the streetlight condition you just sent, with errors related to the invalid message.
-
-# Conclusion
-
-You've learned how to create an AsyncAPI description file and how to generate code from it. Today's sample code is only a bootstrap; you'll need to add your own business logic into it. Take some time to play with it. There are still lots of things to be covered, but the intent of this tutorial is to make it simple for you to get an idea of the potential.
-
-We would love to see what you create with AsyncAPI! As an open-source project, we're open to proposals, questions, suggestions, and contributions. If you don't feel in the mood to contribute but you're using AsyncAPI, just raise your hand by [creating an issue in our Github repo](https://github.com/asyncapi/asyncapi/issues/new) or [join our Slack channel](https://www.asyncapi.com/slack-invite/). Don't be shy.
-
----
-
-<DocsButton
-  suggestions={[
-    {
-      href: '/docs/tutorials',
-      title: 'Tutorials - Overview',
-      type:'back',
-    },
-    {
-      href: '/docs/tutorials/streetlights-interactive',
-      title: 'Streetlights - Interactive (Alpha)',
-      type:'next',
-    }
-  ]}
-/>
\ No newline at end of file
+---

```

Comment:
@starlightknown Draft 2 is looking even better! This is good stuff and I just need to examine it more closely so I can give you detailed feedback. (mostly should be styling guide items that shouldn't be a big deal to change)

I look forward to checking this out, thank you for keeping at it! ⭐  I will tag you again in this PR when I have finished reviewing this latest draft.

Reply:
Would like to recommend you subscribe to the **FREE** plan of **Grammarly**: https://www.grammarly.com/plans

You need to run all your TW drafts through a program like this to catch your first round of grammatical errors, typos, etc. 😄 I promise this will make your life easier! 

<img width="400" alt="Grammarly plans" src="https://user-images.githubusercontent.com/19964402/173464230-90c8fd51-b363-47d5-bb4c-610304aadc01.png">

Reply:
> @starlightknown Draft 2 is looking even better! This is good stuff and I just need to examine it more closely so I can give you detailed feedback. (mostly should be styling guide items that shouldn't be a big deal to change)
> 
> I look forward to checking this out, thank you for keeping at it! star I will tag you again in this PR when I have finished reviewing this latest draft.
> 
> 👉🏽👉🏽 **P.S. Remember that in each `GSoD` PR, we should be adding all 6 other TW interns alongside your GSoD mentors**. Go ahead and add your other 5 teammates to this PR so they can review it too. smiley

I mentioned them in this issue, I was not sure about how to add them as reviewers here

Reply:
@starlightknown ah, so you are saying that it doesn't show you permission to add reviewers, right? Does this gear option to add people not show for you? (if so, thank you for letting me know!)

<img width="300" alt="Screen Shot 2022-06-13 at 7 51 39 PM" src="https://user-images.githubusercontent.com/19964402/173483349-70273e9c-8d78-468b-bef1-51910bfde003.png">

Reply:
yeah, that does not show for us